unity-client/Assets/Scripts/MainScripts/DCL/Controllers/HUD/BuildModeHUD/Scripts/GodMode/TopActionsButtons/TopActionsButtonsController.cs

```diff
@@ -0,0 +1,164 @@
+using System;
+using UnityEngine.EventSystems;
+
+public interface ITopActionsButtonsController
+{
+    event Action OnChangeModeClick,
+                 OnExtraClick,
+                 OnTranslateClick,
+                 OnRotateClick,
+                 OnScaleClick,
+                 OnResetClick,
+                 OnDuplicateClick,
+                 OnDeleteClick,
+                 OnLogOutClick;
+
+    IExtraActionsController extraActionsController { get; }
+
+    void Initialize(ITopActionsButtonsView topActionsButtonsView, ITooltipController tooltipController);
+    void Dispose();
+    void ChangeModeClicked();
+    void ExtraClicked();
+    void TranslateClicked();
+    void RotateClicked();
+    void ScaleClicked();
+    void ResetClicked();
+    void DuplicateClicked();
+    void DeleteClicked();
+    void LogOutClicked();
+    void TooltipPointerEntered(BaseEventData eventData, string tooltipText);
+    void TooltipPointerExited();
+    void SetExtraActionsActive(bool isActive);
+}
+
+public class TopActionsButtonsController : ITopActionsButtonsController
+{
+    public event Action OnChangeModeClick,
+                        OnExtraClick,
+                        OnTranslateClick,
+                        OnRotateClick,
+                        OnScaleClick,
+                        OnResetClick,
+                        OnDuplicateClick,
+                        OnDeleteClick,
+                        OnLogOutClick;
```

Comment:
Not sure if like this styling decision, we don't have anything like this elsewhere in the project and it may be not very merge friendly.

Reply:
I left it because it was already done in this way this in the previous code, but yeah, it makes some noise to me too. I have changed it in order to follow our traditional code style. Changes done in https://github.com/decentraland/explorer/pull/2103/commits/e26a50390bdedff07c960ae34d4c2d95c43daeb0. Thanks for the review Brian!